mhlib/mhConfig.go

```diff
@@ -39,12 +40,13 @@ type MHConfig struct {
 // 4. Command line flags
 // 5. app-specific overrides in MH_CONFIG.
 var DefaultMHConfig = MHConfig{
-	Maintainers:   []string{"none"},
-	PrintRendered: false,
-	Simulate:      false,
-	TargetContext: "localhost",
-	Team:          "sre",
-	SETValues:     []string{""},
+	Maintainers:    []string{"none"},
```

Comment:
I don't understand why so many lines are diffing.  Are there tabs prepending these lines now or were there before?

Reply:
Its not the tab/space leading the lines, but the *vertical alignment* of the `values` of these parameters. It was automatically formatted to keep it that way.

Reply:
Ah ok. I need coffee today, haha. Thanks